### DISCUSSION TITLE
Can you give me a list of possible server-columns?

### DISCUSSION THREAD

**Discussion by x0tester0x** [Dantevg/WebStats#93] [Q&A] [answered]
I need to know what I can configure with the configuration option server-columns.
Can you give me a list off possible server-columns?

**Comment by Dantevg** [comment, marked as answer]
For the `server-columns` list, you can choose from all the column names that you are displaying now. For those columns, their values will then be displayed for a separate row called 'Server', and no longer for the individual players.

As an example, look at the demo page (https://dantevg.nl/mods-plugins/WebStats/demo/). Here you could for example do this: (although im not sure server-columns would work for these columns as they're specifically for players)
```yaml
server-columns: [Mine Diamond, Biome]
```

**Reply by x0tester0x** [reply]
OK, so then when I use `server-columns: [Deaths]` it will sum up all deaths of all players?

**Reply by Dantevg** [reply]
Oh, you're misunderstanding the purpose of the `server-columns` list. WebStats does not do any calculation on those colums. The list is meant for columns displaying statistics that apply to the server as a whole instead of to players. That can be the server's TPS or the number of online players, for instance (see [these placeholders](https://github.com/PlaceholderAPI/PlaceholderAPI/wiki/Placeholders#server) for more).

**Reply by Dantevg** [reply]
If you would like a table footer row that sums up all players' statistics per column, you can open an issue for it (but no promises when or if I will implement it, I've kind of slowed development on WebStats).

**Reply by x0tester0x** [reply]
OK, I did understand now.
